scripts/py_matter_idl/matter_idl/generators/java/ChipEventStructs.jinja

```diff
@@ -26,11 +26,11 @@
       {{encode_tlv(encodable.without_optional(), tag, "opt" + name, depth + 1)}}
     }
   {%- elif encodable.is_list -%}
-      startList({{tag}})
+      startArray({{tag}})
```

Comment:
could you specify the rational for this startArray in PR description?

Reply:
I think we should use array, https://github.com/project-chip/connectedhomeip/blob/94853cfe98c73708a3636163a66d6ffcc22a35d2/src/app/BufferedReadCallback.cpp#L144, the core sdk is using tlv array type for a list of data in terms of attribute or event

Reply:
I updated description. Thanks.